eclair-core/src/test/scala/fr/acinq/eclair/channel/states/e/NormalStateSpec.scala

```diff
@@ -171,6 +181,21 @@ class NormalStateSpec extends TestkitBaseClass with StateTestsHelperMethods {
     alice2bob.expectNoMsg(200 millis)
   }
 
+  test("recv CMD_ADD_HTLC (HTLC dips remote funder below reserve)") { f =>
+    import f._
+    val sender = TestProbe()
+    addHtlc(MilliSatoshi(771000000), alice, bob, alice2bob, bob2alice)
+    crossSign(alice, bob, alice2bob, bob2alice)
+    assert(alice.stateData.asInstanceOf[DATA_NORMAL].commitments.availableBalanceForSend === 40000.msat)
+
+    // actual test begins
+    // at this point alice has minimal amount to sustain a channel (29000 sat ~= alice reserve + commit fee)
```

Comment:
This just to trigger a new build, hopefully semaphore will pick it up this time.

```suggestion
    // at this point alice has the minimal amount to sustain a channel (29000 sat ~= alice reserve + commit fee)
```